doc/reading-list.md

```diff
@@ -86,6 +86,8 @@ Function Shipping
 
 * Function shipping (In-storage-compute) usage in motr: [PDF](PDF/motr_function_shipping.pdf).
 
+  * The layout plan interface allow a generic and fault-tolerant implementation of function shipping: [layout plan](https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h)
+
```

Comment:
The doc is at: 
http://gerrit.mero.colo.seagate.com/plugins/gitiles/mero/+/refs/heads/dev/layout-plan/layout/plan.h

It is seagate internal site, isn't it?
And this is on a development branch.

I don't think it is accessible from outside.

Reply:
The same document is available at GitHub on the following link: 
https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h

I have created the GitHub link so everyone can access the link.

Reply:
Are you sure these two files are the same? Or the contents are similar?
http://gerrit.mero.colo.seagate.com/plugins/gitiles/mero/+/refs/heads/dev/layout-plan/layout/plan.h
https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h

I don't think so.

Reply:
@huanghua78 , I think the only difference is that Mayur changed the license text to Apache.  @TechWriter-Mayur , is that correct?